Makefile

```diff
@@ -14,7 +14,8 @@
 VERSION=11.0.0-dev
 
 DOCKER_IMAGE ?= quay.io/gravitational/teleport
```

Comment:
```suggestion
DOCKER_IMAGE_QUAY ?= quay.io/gravitational/teleport
```

Reply:
Updated in https://github.com/gravitational/teleport/pull/14347/commits/7028c79eb1bb7788641df7b80a1030a88f8dba9c